pkg/providers/amifamily/bootstrap/windows.go

```diff
@@ -0,0 +1,53 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package bootstrap
+
+import (
+	"bytes"
+	"encoding/base64"
+	"fmt"
+	"strings"
+)
+
+type Windows struct {
+	Options
+	ContainerRuntime string
+}
+
+// nolint:gocyclo
+func (w Windows) Script() (string, error) {
+	var userData bytes.Buffer
+	userData.WriteString("<powershell>\n")
+	userData.WriteString("[string]$EKSBootstrapScriptFile = \"$env:ProgramFiles\\Amazon\\EKS\\Start-EKSBootstrap.ps1\"\n")
+	userData.WriteString(fmt.Sprintf(`& $EKSBootstrapScriptFile -EKSClusterName '%s' -APIServerEndpoint '%s'`, w.ClusterName, w.ClusterEndpoint))
+	if w.CABundle != nil {
+		userData.WriteString(fmt.Sprintf(` -Base64ClusterCA '%s'`, *w.CABundle))
+	}
+	if w.ContainerRuntime != "" {
```

Comment:
It looks like you have `w.ContainerRuntime` twice. Any reason to specify it twice? I _think_ you only need to have `w.ContainerRuntime`.

Reply:
My bad. I had a brain fart at that time and wrote the wrong code. :(